xrpl/models/transactions/credential_create.py

```diff
@@ -0,0 +1,81 @@
+"""Model for CredentialCreate transaction type."""
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import (
+    HEX_REGEX,
+    KW_ONLY_DATACLASS,
+    get_credential_type_error,
+    require_kwargs_on_init,
+)
+
+_MAX_URI_LENGTH = 256
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class CredentialCreate(Transaction):
+    """This transaction creates a Credential object. It must be sent by the issuer."""
+
+    account: str = REQUIRED  # type: ignore
+    """
+    The issuer of the credential.
+    """
+
+    subject: str = REQUIRED  # type: ignore
+    """
+    The subject of the credential.
+    """
+
+    credential_type: str = REQUIRED  # type: ignore
+    """
+    A hex-encoded value to identify the type of credential from the issuer.
+    """
+
+    expiration: Optional[int] = None
+    """
+    Optional credential expiration.
+    """
+
+    uri: Optional[str] = None
+    """
+    Optional additional data about the credential (such as a link to the Verifiable
```

Comment:
Remove Optional wording since it's implied in the types.
```suggestion
    The credential expiration.
    """

    uri: Optional[str] = None
    """
    Additional data about the credential (such as a link to the Verifiable
```

Reply:
I'm following the documentation verbatim, I did not write these comments myself